### PR TITLE
Initial commit of AWS Batch setup instructions

### PR DESCRIPTION
**Pull request by bconn-proteinms**

### CODE REVIEW

File: docs/source/set_up_aws.rst
[the remark this answers is not in the document]
Comment (bconn-proteinms) [reply docs/source/set_up_aws.rst:1]: @mriffle nice catch. I fixed it.

**Comment by bconn-proteinms** [comment]
@mriffle I am going to merge this PR unless you have some other changes you would like to me make first.

**Comment by mriffle** [comment]
Did you update the links? If not I can do it after you merge.

**Comment by bconn-proteinms** [comment]
I will make the change to the links and make sure there are no more warnings. Then I will get it merged.

**Pull request merged**